global/wizard.sh

```diff
@@ -96,6 +108,27 @@ esac
 
 setlayout $LAYOUT
 
+echo -e "\n### FORMAT ###"
+
+echo -e "
+   [1] A5
+   [2] A4\n"
```

Comment:
A4 is generated with an asymmetric page layout for exercise and solutions. Handout would need even more adjustments with the fist page looking bad and pagebreaks and pictures needing adjustment. So I would not use it for now, but if it is meant as a preparation I am not against it.

Reply:
We recently added it to the wizard I think more as a future option. The default remains A5 anyways.